templates/regulation/_items.html.twig

```diff
@@ -1,12 +1,16 @@
 {% set duplicateCsrfToken = csrf_token('duplicate-regulation') %}
 {% set deleteCsrfToken = csrf_token('delete-regulation') %}
+{% set displayOrganization = app.user.getOrganizations|length > 1 %}
```

Comment:
(Détail) J'utiliserais un nom de variable avec "is" ou "has" en préfixe

Par ex

```suggestion
{% set isOrganizationShown = app.user.getOrganizations|length > 1 %}
```